magic/loader.py

```diff
@@ -58,13 +58,13 @@ def _lib_candidates_windows():
         "msys-magic-1",
     )
 
-    for i in prefixes:
+    for fname in fnames:
         # libmagic bundled in the wheel
-        yield os.path.join(here, '%s.dll' % i)
+        yield os.path.join(here, '%s.dll' % fname)
         # libmagic in the current working directory
-        yield os.path.join(os.path.abspath('.'), '%s.dll' % i)
+        yield os.path.join(os.path.abspath('.'), '%s.dll' % fname)
         # find_library searches in %PATH% but not the current directory
-        yield find_library(i)
+        yield find_library(fname)
```

Comment:
Should we remove lines 93 and 94 below because a library can be loaded even if it is not a full path?
```python
        if not os.path.exists(lib):
            continue
```

Reply:
ah youre right, the line you commented on is not a full path. @ahupp how about partially reverting https://github.com/ahupp/python-magic/commit/a9e627644efcee42b70cf7634f0ec6d2bab318c8 ?

Reply:
You are correct.  My mistake.